apps/api/src/app/layouts/layouts.module.ts

```diff
@@ -1,13 +1,14 @@
-import { Module } from '@nestjs/common';
+import { forwardRef, Module } from '@nestjs/common';
 
 import { USE_CASES } from './usecases';
 import { LayoutsController } from './layouts.controller';
 
+import { ChangeModule } from '../change/change.module';
 import { MessageTemplateModule } from '../message-template/message-template.module';
 import { SharedModule } from '../shared/shared.module';
 
 @Module({
-  imports: [SharedModule, MessageTemplateModule],
+  imports: [SharedModule, ChangeModule, MessageTemplateModule],
```

Comment:
We need `CreateChange` use case inside Layout module for being able to create all the changes through `CreateLayoutChange`.